src/pybind/mgr/dashboard_v2/HACKING.rst

```diff
@@ -0,0 +1,451 @@
+Dashboard v2 Developer Documentation
+====================================
+
+The Python backend code of this module requires a number of Python modules to be
+installed. They are listed in file ``requirements.txt``.  Using `pip
```

Comment:
s/.  Using/. Using/

There are many other incidents like this across this PR.

Reply:
Replaced all double spaces after periods.